core/src/ephemeral_server/mod.rs

```diff
@@ -217,7 +233,7 @@ impl EphemeralServer {
         if let Some(pid) = self.child.id() {
             let nix_pid = nix::unistd::Pid::from_raw(pid as i32);
             Ok(spawn_blocking(move || {
-                nix::sys::signal::kill(nix_pid, nix::sys::signal::Signal::SIGINT)?;
+                nix::sys::signal::kill(nix_pid, nix::sys::signal::Signal::SIGKILL)?;
```

Comment:
This was breaking on older Python and leaving zombies. Empirical evidence showed on Python this was the Java test server process would get cleaned up properly when running in Python 3.7. I should clarify:

> Do not consider this fixed without running that scenario.

To mean:

> Do not alter this section without running that scenario.

If a Python 3.7 Linux integration test passes with this change, no prob. I do not understand why SIGKILL is leaving zombies and/or not terminating the test server here (or maybe I mistested that scenario or am forgetting which scenarios I tested). I would welcome any assistance debugging it.

Reply:
Oh, weird. OK, I'll change it back.